src/config.ts

```diff
@@ -13,7 +13,7 @@ import {PersonaliaInfo} from "./app/datatyper/personalia";
 import {YtelseDataType} from "./app/datatyper/ytelse";
 
 export type Datasource<T> = () => Promise<Data<T>>;
-export type VisningKomponent<T = {}> = React.ComponentType<{ data: T}> & { placeholder?: React.ComponentType };
+export type VisningKomponent<T = {}> = React.ComponentType<{ data: T}> & { spinner?: React.ComponentType };
```

Comment:
Det er ikke lengre behov for denne typen om man konsekvent skal bruke en spinner.